### ISSUE TITLE
Allow configuration of Faraday.

### ISSUE THREAD

**Issue by geshafer** [AskarZinurov/shift_planning#1]
It'd be helpful if you could configure the gem with a [logger for use with Faraday](https://github.com/AskarZinurov/shift_planning/blob/master/lib/shift_planning/connection.rb#L11) so we could customize it (log level, formatter, options, etc). Perhaps, also provide an option to not register the `Faraday::Logger` middleware at all.

**Comment by AskarZinurov** [comment]
I agree with you, configuration should be more flexible. Will be done soon!
